.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /src/promptflow-core/promptflow/_core/ @microsoft/promptflow-execution
 /src/promptflow-devkit/promptflow/_internal/ @microsoft/promptflow-execution
 /src/promptflow-devkit/promptflow/batch/ @microsoft/promptflow-execution
+/src/promptflow-devkit/promptflow/_proxy/ @microsoft/promptflow-execution
```

Comment:
this folder should be co-own by Python/C# executor and sdk team?

Reply:
updated